packages/coqide/coqide.8.7.1/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "1.2"
+maintainer: "Ralf Jung <post@ralfj.de>"
+authors: "The Coq development team, INRIA, CNRS, University Paris Sud, University Paris 7, Ecole Polytechnique."
+homepage: "https://coq.inria.fr/"
+bug-reports: "https://github.com/coq/coq/issues"
+dev-repo: "https://github.com/coq/coq.git"
+license: "LGPL-2.1"
+build: [
+  [
+    "./configure"
+    "-configdir" "%{lib}%/coq/config"
+    "-prefix" prefix
+    "-mandir" man
+    "-docdir" doc
+    "-libdir" "%{lib}%/coq"
+    "-datadir" "%{share}%/coq"
+    "-usecamlp5"
+    "-camlp5dir" "%{lib}%/camlp5"
+  ]
+  [make "-j%{jobs}%" "coqide-files"]
+  [make "-j%{jobs}%" "coqide-opt"]
+]
+remove: ["rm" "-R" "%{lib}%/coq/ide" "%{doc}%/FAQ-CoqIde"]
+depends: [
+  "camlp5"
```

Comment:
Same question here.

Reply:
Same comment than above.

Reply:
Coq links to camlp5 so indeed if campl5 is updated Coq has to be rebuilt.